src/Security/Http/Logout/ContextInvalidationLogoutHandler.php

```diff
@@ -11,58 +11,33 @@
 
 namespace FOS\HttpCacheBundle\Security\Http\Logout;
 
-use FOS\HttpCache\ProxyClient\Invalidation\BanCapable;
+use FOS\HttpCacheBundle\UserContextInvalidator;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
 use Symfony\Component\Security\Http\Logout\LogoutHandlerInterface;
 
-class ContextInvalidationLogoutHandler implements LogoutHandlerInterface
+/**
+ * @deprecated use ContextInvalidationSessionLogoutHandler in this same namespace as a replacement
+ *
+ * This handler is deprecated since in most cases the session has already been invalidated by the SessionLogoutHandler
```

Comment:
lets be more affirmative: "This handler is deprecated because it never did what it was supposed to do. The session is already invalidated by the SessionLogoutHandler which is always the first logout handler executed."